library/src/main/java/io/constructor/core/ConstructorIo.kt

```diff
@@ -1016,6 +1015,170 @@ object ConstructorIo {
     }
 
     /**
+     * Tracks quiz result click events
+     * ##Example
+     * ```
+     * ConstructorIo.trackQuizResultClick("coffee-quiz", "23AECMA-1EFKCI", "34NCUIEI-214CDN", "shirt-a", "shirt-a--reg", "White shirt", null, null, 10, 1, 10, null);
+     * ```
+     * @param quizId The quiz id
+     * @param quizVersionId version identifier for the quiz. version ID will be returned in quiz request responses
+     * @param quizSessionId session identifier for the quiz. session ID will be returned in quiz request responses
+     * @param customerId The item identifier of the clicked item i.e "PUMP-KAB-0002"
+     * @param variationId The variation item identifier of the clicked item
+     * @param itemName The item name of the clicked item i.e "Jacket Denim"
+     * @param sectionName The section that the results came from, i.e. "Products"
+     * @param resultId The result ID of the quiz response that the selection came from
+     * @param numResultsPerPage The count of quiz results on each page
+     * @param resultPage The current page that quiz result is on
+     * @param resultCount The total number of quiz results
+     * @param actionClass The event class. Leave this blank in mose cases.
```

Comment:
Do we need to expose `actionClass` parameter? I don't think we did for javascript SDKs since we do control it's value internally based on the event type. Is there a case where users will need to change it?

Reply:
🤔 Good point, let me check with the quizzes team on this. I was following the public api documentation.

Reply:
@jjl014 If you happen to know.

Reply:
We made it internal for other implementations for javascript SDKs

Reply:
@stanlp1 That's weird that it's being exposed as a parameter to the client and in our docs. I don't think it's something we want to surface to the consumers since it shouldn't matter to them. I think we should leave it out and remove it from the public docs, but we should check in with the Quizzes team just in case. Thanks!